www-apps/davical/davical-1.1.8.ebuild

```diff
@@ -1,25 +1,26 @@
-# Copyright 1999-2018 Gentoo Authors
+# Copyright 1999-2019 Gentoo Authors
```

Comment:
... I really dont like this update, especially since it has EAPI update and all. You also don't remove 1.1.7 in this commit and leave it in the Manifest file, so after this commit the tree becomes broken (I do see you fix it in the next commit, but if someone syncs the tree to this state...)

Reply:
davical-1.1.7.tar.xz is also for, 1.1.7-r2, but i will reorder the commits in order to properly delete 1.1.7 and 1.1.7-r1 in their own commit.